csrankings-c.csv

```diff
@@ -987,6 +987,7 @@ Chulhee Yun,KAIST,https://chulheeyun.github.io/,Ukl64ggAAAAJ
 Chun Cao,Nanjing University,https://ccao.cc/en,U08cMmAAAAAJ
 Chun Chen 0001,Zhejiang University,http://mypage.zju.edu.cn/chenc,NOSCHOLARPAGE
 Chun Jason Xue,City University of Hong Kong,https://www.cs.cityu.edu.hk/~jasonxue,bVQNPywAAAAJ
+Chun Jiang Zhu,University of North Carolina at Greensboro,https://dblp.org/pid/287/4989.html,0IOSvAQAAAAJ

```

Comment:
The institution name is too long - please abbreviate to UNC - Greensboro (following the format of UNC - Charlotte). DBLP links are also not suitable home page substitutes.